api4/brand_test.go

```diff
@@ -48,7 +47,7 @@ func TestUploadBrandImage(t *testing.T) {
 	} else if resp.StatusCode == http.StatusUnauthorized {
 		CheckUnauthorizedStatus(t, resp)
 	} else {
-		t.Fatal("Should have failed either forbidden or unauthorized")
+		require.FailNow(t, "Should have failed either forbidden or unauthorized")
```

Comment:
`FailNow` ends the execution of also all other tests, which is not ideal. 
```suggestion
		require.Fail(t, "Should have failed either forbidden or unauthorized")
```

Reply:
OK thanks for the feedback,
I was replacing it with something similar to `t.Fatal` which also calls a `FailNow`
Will update this right away